frontend/test/metabase/visualizations/lib/renderer_utils.unit.spec.js

```diff
@@ -123,8 +123,9 @@ describe("getXValues", () => {
       "2019-08-13T00:00:00Z",
     ]);
   });
-  it("should include nulls by default", () => {
-    const xValues = getXValuesForRows([[["foo"], [null], ["bar"]]]);
+  it("should include nulls for ordinal", () => {
```

Comment:
@mazameli @flamber is it correct that we should show nulls only on the ordinal axis? I saw many multiple comments in code that say so but the tests expect it to be the default behavior.